packages/shared/src/components/modals/NewCommentModal.tsx

```diff
@@ -265,7 +285,12 @@ export default function NewCommentModal({
 
   return (
     <ResponsiveModal
-      {...{ contentRef: modalRef, onRequestClose: confirmClose, ...props }}
+      {...{
+        contentRef: modalRef,
+        onRequestClose: confirmClose,
+        style: { content: { overflow: 'initial' } },
```

Comment:
Do we need the overflow initial here? 
Could we not use `overflow-auto` here?

Reply:
In the past, we can't. I have now introduced a change that makes this unnecessary anymore 🚢